### PR TITLE
Fixing a typo in Non_English_Downstream_Tasks (NER) tutorial

### PR DESCRIPTION
**Pull request by pasandi20**

### CODE REVIEW

**Comment by ericharper** [comment]
Can you make the base branch `r1.3.0`?

File: tutorials/nlp/Non_English_Downstream_Tasks_(NER).ipynb [hunk 1]
```diff
@@ -1 +1,880 @@
...
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "OETcTQlcguCm"
+      },
+      "source": [
+        "BRANCH = 'main'"
```
Comment (ericharper) [inline tutorials/nlp/Non_English_Downstream_Tasks_(NER).ipynb:33]: `main` should be `r1.3.0`

**Review by ericharper — CHANGES_REQUESTED** [review]
This is a bug fix for the r1.3.0 release. Please change the base branch of the PR and also change the pinned branch for the notebook.

**Comment by ekmb** [comment]
@ericharper @pasandi20 is this PR still needed?

**Comment by okuchaiev** [comment]
looks like this became obsolete. closing

**Pull request closed without merging**